content/_index.zh-hant.md

```
@@ -0,0 +1,63 @@
+2. **與 CI/CD 整合**：透過一致的 branch 命名，它可以協助自動化系統（例如 Continuous Integration/Continuous Deployment 管線）根據 branch 類型觸發特定動作（例如，從 release branch 自動部署）。
+3. **團隊協作**：藉由明確標示 branch 的目的，促進團隊協作、降低誤解，並讓成員更容易在不同任務間切換而不致混淆。
+
+## 規範
+
+### branch 命名前綴
+
+此 branch 規範支援以下前綴，並應採用如下結構：
+
+---
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)